src/manipulation.js

```diff
@@ -202,7 +202,11 @@ function domManip( collection, args, callback, ignored ) {
 								jQuery._evalUrl( node.src );
 							}
 						} else {
-							DOMEval( node.textContent.replace( rcleanScript, "" ), doc );
+							attr = {};
+							if ( node.hasAttribute && node.hasAttribute( "nonce" ) ) {
+								attr.nonce = node.getAttribute( "nonce" );
+							}
+							DOMEval( node.textContent.replace( rcleanScript, "" ), doc, attr );
```

Comment:
In the name of [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it), it's probably better to use `node.getAttribute( "nonce" )` instead of an attribute map.

Reply:
So should we just pass `nonce` as a third arg to `DOMeval()`? I agree we don't need this generality yet.

Reply:
I've implemented the requested changes. One thought I had was is it worth passing `node` instead of `nonce` into `DOMeval()` as may be a more useful 3rd argument in future whilst better grouping relevant code and eliminating the additional local variable (`nonce`) within `domManip()`. 

Also my apologies about the delayed response. I've been quite unwell recently.